apps/build-onchain-apps/contracts/script/SignatureMintERC721.s.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {Script, console2} from "forge-std/Script.sol";
+import "../src/CustomERC1155.sol";
+import "../src/SignatureMintERC721.sol";
+
+contract SignatureMintERC721Script is Script {
+    function setUp() public {}
+
+    function run() public {
+        uint256 privateKey = vm.envUint("PRIVATE_KEY");
+        address signer = vm.envAddress("SIGNATURE_MINT_SIGNER");
```

Comment:
Wonder if `vm.envAddress` converts the address into the checksum address or do we need to it that? 

One user could potentially provide us an address in all lowercase and we might have to convert it to a checksum address is vm.envAddress doesnt do that for us

Reply:
this converts into an address object under the covers, so we should be good.  Tested locally and works as expected:
![image](https://github.com/coinbase/build-onchain-apps/assets/7040142/5a394546-25d7-4e8f-9288-e73d324be044)